internal/command/logs/logs.go

```diff
@@ -50,9 +50,11 @@ Use --no-tail to only fetch the logs in the buffer.
 		flag.Region(),
 		flag.JSONOutput(),
 		flag.String{
-			Name:        "instance",
-			Shorthand:   "i",
-			Description: "Filter by instance ID",
+			Name:              "machine",
+			Shorthand:         "m",
```

Comment:
Should we add `-m` to `fly ssh console` to keep things consistent?

Reply:
I've just noticed, that there are two more place where the "machine" flag is declared, "internal/command/console/console.go", "internal/command/registry/vulns.go".
These are all of the places so far with the "machine" flag mentioned (prior to inclusion of the flag in logs):

1. [internal/command/console/console.go](https://github.com/superfly/flyctl/blob/master/internal/command/console/console.go#L49)
2. [internal/command/ssh/console.go](https://github.com/superfly/flyctl/blob/master/internal/command/ssh/console.go#L43)
3. [internal/command/registry/sbom.go](https://github.com/superfly/flyctl/blob/master/internal/command/registry/sbom.go#L39)
4. [internal/command/registry/vulns.go](https://github.com/superfly/flyctl/blob/master/internal/command/registry/vulns.go#L45)

All of the places above, with the exception of number 4, somehow do not have any shorthand for  their "machine" flags. @btoews Do you think there might  be a reason for this? I've just removed the shorthand "m" I've declared for logs.go, just in case there's any particular reason why this shorthand is not included for the others, and to also follow consistency with the other commands above.

Reply:
I'm not sure what the philosophy is or if there even is one. It might be that having a shorthand and normal option feels like adding two more CLI flags, which feels like too much when there are a lot of flags on some of these commands. It might be a concern about conflicting with other flags. It might be that people don't expect it to be used enough to warrant a shorthand.

Personally, I use the `--machine`/`--instance` flags a lot and would appreciate having a `-m` flag wherever possible...

Reply:
ok! Thanks a lot for another round of great insight @btoews ! 

Gonna merge this within the day, I'ma try to pass all the tests that failed